src/components/Pages/Unit/widgets/UnitMaterials/Transcription/Transcription.js

```diff
@@ -139,9 +162,21 @@ class Transcription extends Component {
     this.setState({ selectedFile, language: newLanguage });
   };
 
+  prepareScrollToSearch = (data, search) => {
+    const result = data.split('<p').map(p => {
+      const clearTags = p.replace(/<.+?>/gi, '');
+      if (clearTags.indexOf(search) === -1) {
+        return p;
+      }
+      return ` class="scroll-to-search"  id="${SCROLL_SEARCH_ID}" ${p}`;
```

Comment:
All paragraphs will contain the same id?

Reply:
Maybe to split by search result, to add class and id to the first one and to join back?

Reply:
it must be just one selected paragraph for page